packages/clerk-js/src/ui/components/OrganizationProfile/__tests__/InviteMembersPage.test.tsx

```diff
@@ -65,6 +202,17 @@ describe('InviteMembersPage', () => {
             createdAt: new Date(),
             updatedAt: new Date(),
           },
+          {
+            pathRoot: '',
+            reload: jest.fn(),
+            id: 'admin',
+            key: 'admin',
+            name: 'Admin',
+            description: '',
+            permissions: [],
```

Comment:
Those tests were mocking with a single role, therefore initiating the field with that one instead and the assertion for "Select role" started to fail.